example_data/sample_bundle.json

```
@@ -20,12 +30,17 @@
             "text": "NHS UK Healthcare Provider"
           }
         ],
-        "name": "Great Ormond Street Hospital for Children NHS Foundation Trust",
+        "name": "North London Lab 1",
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "2078298870"
+          }
+        ],
         "address": [
           {
             "line": [
-              "Level 4 Barclays House",
-              "37 Queens Square"
+              "Levels 4-6 Barclay House, 37 Queen Square"
```

Should be using lines here
```suggestion
              "Levels 4-6 Barclay House", 
              "37 Queen Square"
```